### PR TITLE
Update pkgs

### PR DESCRIPTION
**Pull request by SimHoZebs**
More than a handful of packages are outdated. Expo, especially, is a major version behind.
```
Package                              Current   Wanted   Latest  Location                                          Depended by
@babel/core                           7.23.2   7.23.7   7.23.7  node_modules/@babel/core                          frontend@1.0.0
@emotion/react                       11.11.1  11.11.3  11.11.3  node_modules/@emotion/react                       frontend@1.0.0
@fortawesome/fontawesome-svg-core      6.4.2    6.5.1    6.5.1  node_modules/@fortawesome/fontawesome-svg-core    frontend@1.0.0
@fortawesome/free-regular-svg-icons    6.4.2    6.5.1    6.5.1  node_modules/@fortawesome/free-regular-svg-icons  frontend@1.0.0
@prisma/client                         5.4.2    5.8.1    5.8.1  node_modules/@prisma/client                       frontend@1.0.0
@types/node                          20.10.3  20.11.5  20.11.5  node_modules/@types/node                          MuHASS       
axios                                  1.5.1    1.6.5    1.6.5  node_modules/axios                                frontend@1.0.0
expo                                 49.0.16  49.0.22   50.0.2  node_modules/expo                                 frontend@1.0.0
expo-network                           5.4.0    5.4.0    5.8.0  node_modules/expo-network                         frontend@1.0.0
expo-status-bar                        1.6.0    1.6.0   1.11.1  node_modules/expo-status-bar                      frontend@1.0.0
metro-runtime                         0.79.1   0.79.1   0.80.4  node_modules/metro-runtime                        frontend@1.0.0
nodemon                                3.0.1    3.0.3    3.0.3  node_modules/nodemon                              backend@npm:server@1.0.0
prisma                                 5.4.2    5.8.1    5.8.1  node_modules/prisma                               frontend@1.0.0
react-native                          0.72.6   0.72.6   0.73.2  node_modules/react-native                         frontend@1.0.0
styled-components                     5.3.11   5.3.11    6.1.8  node_modules/styled-components                    frontend@1.0.0
typescript                             5.3.2    5.3.3    5.3.3  node_modules/typescript                           MuHASS   
```
This leaves us with vulnerabilities - 31 vulnerabilities (19 moderate, 12 high), according to NPM - and requires us to access older documentation to work with.

The plan is to update packages and update our code if any breaking changes necessitate it.

### CODE REVIEW

**Comment by SimHoZebs** [comment]
Up to a140b28, I detected no breaking changes and needed an update. The following packages still need upgrade.

```
Package            Current   Wanted   Latest  Location                        Depended by
@babel/core         7.23.2   7.23.7   7.23.7  node_modules/@babel/core        frontend@1.0.0
@emotion/react     11.11.1  11.11.3  11.11.3  node_modules/@emotion/react     frontend@1.0.0
axios                1.5.1    1.6.5    1.6.5  node_modules/axios              frontend@1.0.0
expo               49.0.16  49.0.22   50.0.2  node_modules/expo               frontend@1.0.0
expo-network         5.4.0    5.4.0    5.8.0  node_modules/expo-network       frontend@1.0.0
expo-status-bar      1.6.0    1.6.0   1.11.1  node_modules/expo-status-bar    frontend@1.0.0
metro-runtime       0.79.1   0.79.1   0.80.4  node_modules/metro-runtime      frontend@1.0.0
react-native        0.72.6   0.72.6   0.73.2  node_modules/react-native       frontend@1.0.0
styled-components   5.3.11   5.3.11    6.1.8  node_modules/styled-components  frontend@1.0.0
```

**Comment by promise310** [comment]
> The following packages should be updated for best compatibility with the installed expo version:
>   expo@50.0.6 - expected version: ~50.0.11
> Your project may not work correctly until you install the correct versions of the packages.

I got this message when I ran the code. But I can still open it at the expo. 
I tried to use "expo upgrade" the Expo CLI to update the Expo package. And I got this warning. 

> warning: Watchman `watch-project` returned a warning: Recrawled this watch 35 times, most recently because:
> MustScanSubDirs UserDroppedTo resolve, please review the information on
> https://facebook.github.io/watchman/docs/troubleshooting.html#recrawl
> To clear this warning, run:
> `watchman watch-del '/Users/seoyoung/Desktop/MuHASS' ; watchman watch-project '/Users/seoyoung/Desktop/MuHASS'`


This warning is related to Watchman, a file watching service used by development tools such as React Native. I'm not sure if the Watchman warning is only for my laptop environment.
I used these commands to clear the warning. 

> watchman watch-del '/Users/seoyoung/Desktop/MuHASS'
> watchman watch-project '/Users/seoyoung/Desktop/MuHASS'

File: package.json
Comment (SimHoZebs) [inline package.json:1]: did `npx expo upgrade` add these packages? If expo being outdated was the issue, the expo package in frontend workspace should've updated instead.

**Comment by SimHoZebs** [comment]
I just committed additional package updates that should get rid of the error message regarding the `expo` version and removed the react-native dependency on the monorepo scope. Please see if this works and let me know if this should be reverted.

> This warning is related to Watchman, a file watching service used by development tools such as React Native. I'm not sure if the Watchman warning is only for my laptop environment.

I can not reproduce this in my environment. It might be specific to macOS. Could you add this information to our README.md for future reference?

**Comment by promise310** [comment]
It works well. I think we can merge it into the main. 
When I run it now, I don't get a Watchman warning. But I can still add about the warning in the README.md just in case.

**Pull request merged**